Source/Charts/Data/Implementations/Standard/ChartDataSet.swift

```diff
@@ -68,23 +75,7 @@ open class ChartDataSet: ChartBaseDataSet
     
     /// minimum x-value in the value array
     internal var _xMin: Double = Double.greatestFiniteMagnitude
-    
-    /// *
-    /// - note: Calls `notifyDataSetChanged()` after setting a new value.
-    /// - returns: The array of y-values that this DataSet represents.
-    @objc open var values: [ChartDataEntry]
-    {
-        get
-        {
-            return _values
-        }
-        set
-        {
-            _values = newValue
-            notifyDataSetChanged()
-        }
-    }
-    
+
     /// Use this method to tell the data set that the underlying data has changed
     open override func notifyDataSetChanged()
```

Comment:
It seems we can safely remove `open override func notifyDataSetChanged()`? Since its superclass `ChartBaseDataSet`->`notifyDataSetChanged()` already called `notifyDataSetChanged()`

Reply:
You are correct!

Reply:
what you think? It's ok either keep or remove.

Reply:
I'm removing it now

Reply:
Done